test/gtest/common/test_ucc.cc

```diff
@@ -501,6 +502,12 @@ const std::vector<UccTeam_h> &UccJob::getStaticTeams()
                                std::end(staticTeamSizes));
     if (0 == staticTeams.size()) {
         for (auto ts : teamSizes) {
+            if (ts == 1 && !tl_self_available()) {
+                /* don't use team_size = 1 if there is no tl/self.
+                   we can't modify nStaticTeams, so just use some other
+                   team_size */
+                ts = 3;
```

Comment:
Can't we just skip if ts is 1? (i.e., do `continue` here)

Reply:
no, as comment says above, tests assume there nStaticTeams available, so we need to create all of them